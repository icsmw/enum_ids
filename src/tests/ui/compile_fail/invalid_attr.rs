use enum_ids::enum_ids;

#[enum_ids(unknown = "value")]
pub enum Kind {
    A(i32),
    B { value: String },
    C,
}

fn main() {
    let _ = Kind::A(1);
}
