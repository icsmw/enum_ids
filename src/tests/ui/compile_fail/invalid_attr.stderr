error: Cannot parse attribute "unknown": Unknown attribute "unknown"
 --> src/tests/ui/compile_fail/invalid_attr.rs:3:12
  |
3 | #[enum_ids(unknown = "value")]
  |            ^^^^^^^

error[E0433]: failed to resolve: use of undeclared type `Kind`
  --> src/tests/ui/compile_fail/invalid_attr.rs:11:13
   |
11 |     let _ = Kind::A(1);
   |             ^^^^ use of undeclared type `Kind`
